### DISCUSSION TITLE
Auto calculated width and hieght using perntageWidth

### DISCUSSION THREAD

**Discussion by 7ory** [dlemstra/Magick.NET#1135] [Help] [answered]
Is there a way I can auto calculate the height and width to fit images that are longer and shorter, avoiding small images from getting stretched out 
![image](https://user-images.githubusercontent.com/50345453/154096118-9c314851-c5b4-464a-871b-8687ba2fb744.png)

![image](https://user-images.githubusercontent.com/50345453/154095961-89b6fc28-2c11-4ef7-9a2c-3b01076aa870.png)

Here is the repository https://github.com/7ory/ImageCompressionAPI

**Comment by dlemstra** [comment, marked as answer]
The issue is that you are setting `IgnoreAspectRatio` to `true`. When you set the size to 480x2460 the image will be fit into the specified space.

p.s. You might also want to take a look at the `ImageOptimizer` that can be used to reduce the size of your jpeg file.

**Reply by 7ory** [reply]
Alright, thank you man :)

How does **ImageOptimizer** work?

**Reply by dlemstra** [reply]
It will reduce thw size of the output file without quality loss.
